.github/workflows/tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Test Splitter"
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [main]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9, 3.11]
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4

```

Comment:
there's already a `5.x` series GH action available for `actions/setup-python`: https://github.com/actions/setup-python/releases/tag/v5.1.1

Reply:
bumped!